### ISSUE TITLE
Add additional models

### ISSUE THREAD

**Issue by rabernat** [pyqg/pyqg#36]
Eady, stacked eady, multi-layer, etc. See  Joern Callies' qg model.
https://github.com/joernc/QGModel/blob/master/model.py

**Comment by crocha700** [comment]
I'm picking up the multi-layer model.

@rabernat, given that this is for 0.2, should I branch off from the development branch? Do we have a new development brach?

**Comment by rabernat** [comment]
I have gotten rid of the development branch, since now we have stable versions available through releases:
https://github.com/pyqg/pyqg/releases

Master now contains the most recent code. Now that we have the travis-ci set up, we should only be merging working PR's. So master should always be usable.

What do you think about that approach? Still trying to figure out the best way to do it.

**Comment by crocha700** [comment]
This approach seems to work very well. I think we would like to have an unstable version under develop or other branch for big additions modifications (maybe not necessary for 0.2, but for 1.0?)

**Comment by rabernat** [comment]
@crocha700 are you going to follow Joern's approach of making the two-layer model just the n=2 case of the multi-layer model?

**Comment by crocha700** [comment]
Yes, I think  the best approach is to generalize the qg_model.py.

**Comment by crocha700** [comment]
I just found a "eady_model.py" in my fork. I will need to refactor this to be consistent with current versions of pyqg. So, no need for other people to work this.

**Comment by crocha700** [comment]
@rabernat shall we create a develop branch for 0.2? I think it would be nice to have an unstable branch. I've worked on the multi-layer implementation (also includes stability analysis and topography) and the eady model .  It is rather unstable and it would be nice to have you guys test/review it...

**Comment by rabernat** [comment]
What would be the downside to using the master branch?

Keep in mind that we have no "users" to speak of. Plus we have a stable release available through pip.

**Comment by crocha700** [comment]
Well, suppose we find a bug in current version and we want to fix it and release an updated version before we release these enhancements?

**Comment by rabernat** [comment]
Of course it's totally fine with me.
